content/docs/rendering-elements.md

```diff
@@ -1,75 +1,77 @@
 ---
 id: rendering-elements
-title: Rendering Elements
+title: رندر شدن المان‌ها
```

Comment:
@zahrajoulaei I could not find any persian word that conveys all the meaning like "render"!

Reply:
I think render is really fine. At least that's what I think. I think it's like there are words that shouldn't be translated because they have specific meaning in react context. An M.D student is trained to get used to hearing common medical phrases from the beginning no matter where they live. The best we can do is to link these words to their explanation in glossary. Like reactjs.org does.

Reply:
yes me too! then it's fine from my side :)